tests/src/python/test_provider_oracle.py

```diff
@@ -819,6 +822,15 @@ def testEvaluateDefaultValues(self):
         attributes = [feat.attributes() for feat in vl.getFeatures()]
         self.assertEqual(attributes, [[1, 'qgis'], [2, 'test'], [3, 'qgis'], [4, 'test']])
 
+    def testCreateEmptyLayer(self):
+        uri = self.dbconn + "srid=4326 type=POINT table=\"EMPTY_LAYER\" (GEOM)"
+        exporter = QgsVectorLayerExporter(uri=uri, provider='oracle', fields=QgsFields(), geometryType=QgsWkbTypes.Type.Point, crs=QgsCoordinateReferenceSystem(4326), overwrite=True)
+        self.assertEqual(exporter.errorCount(), 0)
+        self.assertEqual(exporter.errorCode(), 0)
+        # cleanup
```

Comment:
Could we check if the according table has been created inside the database ?

Reply:
@troopa81 you mean before or after the calling the QgsVectorLayerExporter-constructor? Or both?
I have thought about checking before but concluded that that is always a fresh docker instance of db, so no need to check if it is already there. And I use 'overwrite=True', in any case...
So probably you mean after :-)

Reply:
Yes, I mean after essentially.